decision_maker.go

```diff
@@ -29,7 +29,7 @@ func makePlan(s *state) *plan {
 func decide(r *release, s *state) {
 	if destroy {
 		if ok, rs := helmReleaseExists(r, ""); ok {
-			deleteRelease(r, rs)
+			deleteRelease(r, rs, s)
```

Comment:
I think that passing the whole state here is a bit overkill. Maybe it would be better to create a global variable in the init stage or something like that.

Reply:
Just realized that the global settings variable exists. The tillerless variable already exists in global state. I just need to use it.